lib/concurrent_guest_installations.pm

```diff
@@ -206,7 +206,7 @@ sub junit_log_provision {
         $_guest_installations_results->{$_}{stop_run}  = ($guest_instances{$_}->{stop_run} eq '' ? time() : $guest_instances{$_}->{stop_run});
         $_guest_installations_results->{$_}{test_time} = strftime("\%Hh\%Mm\%Ss", gmtime($_guest_installations_results->{$_}{stop_run} - $_guest_installations_results->{$_}{start_run}));
     }
-    $self->{"product_tested_on"} = script_output("cat /etc/issue | grep -io \"SUSE.*\$(arch))\"");
+    $self->{"product_tested_on"} = script_output("cat /etc/issue | grep -io -e \"SUSE.*\$(arch))\" -e \"openSUSE.*Tumbleweed.*[0-9]\"");
```

Comment:
You can make this part more general. Remove 'tumbleweed' only keep 'opensuse'.

Reply:
Yes. then 'openSUSE Leap xx' is included. thanks.